src/features/channels/components/channel-list/create-channel/CreateButton.tsx

```diff
@@ -36,6 +37,10 @@ const CreateButton = () => {
   };
 
   const createNewChannel = (data: ChannelDTO) => {
+    if (!data.channelName.trim()) {
```

Comment:
Is channelName prop not nullable? are we always going to get channelName prop filled in with a value?

Reply:
Yes, due to linked with an input which value only can be a string